doc/setting_up_sidewalk_environment/setting_up_sidewalk_gateway.rst

```diff
@@ -1,81 +1,25 @@
-******************************************
-
-If you are setting up a new Echo device, complete the setup through the Alexa app.
-For an existing Echo device, go to the :ref:`setting_up_sidewalk_gateway_new_device` section.
-
-#. Download the `Alexa app`_ and open it.
-#. Click :guilabel:`More`, and select :guilabel:`Add device`.
-#. Plug in your device.
-#. Follow the instructions in your mobile app.
-#. Click :guilabel:`Enable` in the landing page.
-
-   .. figure:: /images/sidewalk_alexa_image1.png
-
-.. _setting_up_sidewalk_gateway_new_device:
-
-Enabling Sidewalk on an existing Echo device
-********************************************
-
-If you already have an existing Echo device, check the device settings to determine whether your device can act as a Sidewalk Gateway.
-
-#. Open :guilabel:`More`, and select :guilabel:`Settings`.
-#. Click :guilabel:`Account Settings`.
-#. Select :guilabel:`Amazon Sidewalk` and enable it.
-
-   .. figure:: /images/sidewalk_alexa_image2.png
-
-.. _setting_up_sidewalk_gateway_linking_ring_amazon_account:
-
-Linking your Ring account to your Amazon account
-************************************************
-
-You can link your Ring account to your Amazon account using the Ring app.
-
-#. Open the Ring page in `App store`_ (iOS) or `Play store`_ (Android).
-#. Download and install the Ring app on your mobile device.
-#. Create a new account, or log into your existing Ring account.
-#. Open the Ring application.
-#. Log into your account.
-   If you do not have a Ring account, create it:
-
-   a. Select :guilabel:`Create Account`.
-   #. Follow the instructions provided in the application.
-   #. Click :guilabel:`Enable` in the landing page.
-
-   .. figure:: /images/sidewalk_ring_image1.png
-
-#. Click the menu button, and navigate to :guilabel:`Account Settings`.
-#. Click :guilabel:`Link your Amazon account` and then :guilabel:`Link Accounts`.
-#. Enter the password for your Ring account.
-#. Log into your Amazon Account you used to provision your Sidewalk Endpoint in the Sidewalk console.
-#. Click :guilabel:`Sign in`.
-   Your accounts are now linked.
-
-   .. figure:: /images/sidewalk_ring_image2.png
+For the exact instructions, refer to the `Setting up a Sidewalk gateway`_ documentation.
 
 .. note::
    If you have multiple Echo devices registered under a single Amazon account, only one of those devices will be able to operate as an FSK gateway even if other devices have the same capability.
    If you need to operate multiple FSK gateways, we recommend setting them up under separate Amazon accounts.
 
+.. TBC: "Linking to your ring account" was included in our documentation but does not appear after Amazon's "setting up a Sidewalk gateway". Should the instruction be kept on our side, or should we refer to this instead - https://docs.sidewalk.amazon/mobile-sdk/developer-guide/sidewalk-mobile-sdk-overview/#what-is-amazon-sidewalk-mobile-sdk? Or maybe drop it completely?
```

Comment:
@wojciechpluszynski - question to this section (line 44 in the previous file)

Reply:
This step is not required when setting up new Echo device and Sidewalk end device. Let's remove from Nordic documentation.